app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -30,10 +30,18 @@ def update
         end
       end
 
+      def remove_exemption
+        @enrollment = Enrollment.find(params[:id])
```

Comment:
I'd move it into a `set_enrollment` method at the first time it was needed in another method. Also I avoided this approach here as there is already the `enrollment` method doing the same job in this controller.

All in all - this is further reason for moving the functionality into another controller.